secretariat/migrations/0004_remove_old_default_uuid.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 4.2.5 on 2023-10-19 09:25
+# @see https://docs.djangoproject.com/en/4.2/topics/migrations/#data-migrations
+
+from django.db import migrations
+
+
+def remove_old_default_uuid(apps, _):
```

Comment:
La migration suivante ne passe pas si la base contient des usagers dont le `outline_uuid` n'est pas un uuid valide. Or la chaîne "not a valid uuid", comme indiqué, n'est pas un uuid valide.
Par contre, `None` ça passe.

Reply:
C'est si propre. Moi j'avais pensé faire corriger / supprimer à la main les quelques users qu'on avait créé. mais c'est mieux en propre et en plus j'apprends un truc 👍🏽